contracts/src/libraries/EigenDAHasher.sol

```diff
@@ -14,31 +14,28 @@ library EigenDAHasher {
      * @notice hashes the given metdata into the commitment that will be stored in the contract
      * @param batchHeaderHash the hash of the batchHeader
      * @param signatoryRecordHash the hash of the signatory record
-     * @param fee the fee paid in paymentToken for the batch
      * @param blockNumber the block number at which the batch was confirmed
      */
     function hashBatchHashedMetadata(
         bytes32 batchHeaderHash,
         bytes32 signatoryRecordHash,
-        uint96 fee,
         uint32 blockNumber
     ) internal pure returns(bytes32) {
-        return keccak256(abi.encodePacked(batchHeaderHash, signatoryRecordHash, fee, blockNumber));
+        return keccak256(abi.encodePacked(batchHeaderHash, signatoryRecordHash, blockNumber));
     }
 
     /**
      * @notice hashes the given metdata into the commitment that will be stored in the contract
      * @param batchHeaderHash the hash of the batchHeader
-     * @param fee the fee paid in paymentToken for the batch
+     * @param confirmationData the confirmation data of the batch
      * @param blockNumber the block number at which the batch was confirmed
      */
     function hashBatchHashedMetadata(
         bytes32 batchHeaderHash,
-        bytes memory nonSignerStakesAndSignature,
-        uint96 fee,
+        bytes memory confirmationData,
         uint32 blockNumber
     ) internal pure returns(bytes32) {
-        return keccak256(abi.encodePacked(batchHeaderHash, fee, blockNumber));
+        return keccak256(abi.encodePacked(batchHeaderHash, confirmationData, blockNumber));
```

Comment:
i think we should hash the confirmation data so we don't need to repost it

`keccak256(abi.encodePacked(batchHeaderHash, keccak256(confirmationData), blockNumber))`